2022-06-15 17:20:33,833 INFO    MainThread:49160 [wandb_setup.py:_flush():76] Loading settings from /Users/zhangqiming/.config/wandb/settings
2022-06-15 17:20:33,833 INFO    MainThread:49160 [wandb_setup.py:_flush():76] Loading settings from /Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/wandb/settings
2022-06-15 17:20:33,833 INFO    MainThread:49160 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2022-06-15 17:20:33,834 INFO    MainThread:49160 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program_relpath': 'project/exmaple.py', 'program': '/Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/exmaple.py'}
2022-06-15 17:20:33,834 INFO    MainThread:49160 [wandb_init.py:_log_setup():428] Logging user logs to /Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/wandb/run-20220615_172033-tkh30f3j/logs/debug.log
2022-06-15 17:20:33,834 INFO    MainThread:49160 [wandb_init.py:_log_setup():429] Logging internal logs to /Users/zhangqiming/Desktop/RL_research/Reinforcement-Learning/project/wandb/run-20220615_172033-tkh30f3j/logs/debug-internal.log
2022-06-15 17:20:33,834 INFO    MainThread:49160 [wandb_init.py:init():462] calling init triggers
2022-06-15 17:20:33,834 INFO    MainThread:49160 [wandb_init.py:init():466] wandb.init called with sweep_config: {}
config: {}
2022-06-15 17:20:33,835 INFO    MainThread:49160 [wandb_init.py:init():515] starting backend
2022-06-15 17:20:33,835 INFO    MainThread:49160 [backend.py:_multiprocessing_setup():99] multiprocessing start_methods=fork,spawn,forkserver, using: spawn
2022-06-15 17:20:33,857 INFO    MainThread:49160 [backend.py:ensure_launched():217] starting backend process...
2022-06-15 17:20:33,878 INFO    MainThread:49160 [backend.py:ensure_launched():222] started backend process with pid: 49169
2022-06-15 17:20:33,883 INFO    MainThread:49160 [wandb_init.py:init():525] backend started and connected
2022-06-15 17:20:33,909 INFO    MainThread:49160 [wandb_init.py:init():596] updated telemetry
2022-06-15 17:20:33,973 INFO    MainThread:49160 [wandb_init.py:init():628] communicating run to backend with 30 second timeout
2022-06-15 17:20:35,332 INFO    MainThread:49160 [wandb_run.py:_on_init():1924] communicating current version
2022-06-15 17:20:35,348 WARNING MainThread:49160 [wandb_init.py:init():1020] interrupted
Traceback (most recent call last):
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 999, in init
    run = wi.init()
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/wandb/sdk/wandb_init.py", line 660, in init
    run._on_init()
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/wandb/sdk/wandb_run.py", line 1926, in _on_init
    current_version=wandb.__version__
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/wandb/sdk/interface/interface.py", line 105, in communicate_check_version
    ret = self._communicate_check_version(check_version)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 430, in _communicate_check_version
    result = self._communicate(rec)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/wandb/sdk/interface/interface_shared.py", line 226, in _communicate
    return self._communicate_async(rec, local=local).get(timeout=timeout)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/site-packages/wandb/sdk/interface/router.py", line 37, in get
    is_set = self._object_ready.wait(timeout)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/threading.py", line 552, in wait
    signaled = self._cond.wait(timeout)
  File "/Users/zhangqiming/opt/anaconda3/envs/LtC/lib/python3.7/threading.py", line 300, in wait
    gotit = waiter.acquire(True, timeout)
KeyboardInterrupt
